Thoughtless-Eels/Bangazon-WebApp
# removed file from tracking

## Link to Ticket
Ticket: # [Link]()

## Description of Proposed Changes
-I went into vim and untracked the app settings file 
-I connected a successful string connection from my Visual Studio to the SQL Database
My computer uses SQL Server Express so I need a different connection string than my other teammates 
- 

## Steps to Test

Outline the steps to test
Only my computer uses SQL Server Express so that is not something another teammate can check but make sure your gitignore is untracked in the appsettings file
```sh
git fetch --all
git checkout <feature_branch>

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code
At least one teammate needs to check this pull request.
## Definition of Done
Everyone's gitignore is not being tracked in the app settings file
1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
